automation/README.md

```diff
@@ -0,0 +1,67 @@
+# Automation with oVirt CI
+
+In order to keep playbooks in this repository operational,
+it is being integrated with [oVirt CI System][ovirt-ci-system-doc].
+Everything located under `automation` directory is related to integration
+to [oVirt CI System][ovirt-ci-system-doc].
```

Comment:
`is related to integration to oVirt CI System` and therefore this README.md should describe how that oVirt CI System works, not playbooks.

Reply:
Addressed